tests/tailor.js

```diff
@@ -361,30 +382,11 @@ describe('Tailor', () => {
                 .then(response => {
                     assert.equal(
                         response.headers.link,
-                        '<http://localhost:8080>; rel="preload"; as="style"; nopush,<http://localhost:8080>; rel="preload"; as="script"; nopush; '
+                        '<http://loader>; rel="preload"; as="script"; nopush; crossorigin,<http://localhost:8080>; rel="preload"; as="style"; nopush;,<http://localhost:8080>; rel="preload"; as="script"; nopush;'
                     );
                 })
                 .then(done, done);
         });
-
-        it('"shouldn\'t send preloading headers if primary fragment doesn\'t exist"', done => {
```

Comment:
test covered in https://github.com/zalando/tailor/pull/203/files#diff-67c49567f86ec6fe981f9b76efc15c75R319